mne/time_frequency/tests/test_multitaper.py

```diff
@@ -3,7 +3,7 @@
 from numpy.testing import assert_array_almost_equal
 from distutils.version import LooseVersion
 
-from mne.time_frequency import dpss_windows, multitaper_psd
+from mne.time_frequency.multitaper import dpss_windows, _psd_multitaper
```

Comment:
I'm still +1 on using the public-facing function. Our tests should prefer using public-facing interfaces as much as possible. Since your public function is currently mostly a wrapper to the private one, there seems to be no harm in using it.